x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/documents.tsx

```diff
@@ -32,12 +54,15 @@ export const Documents: React.FC<Props> = ({ engineBreadcrumb }) => {
             <h1>{DOCUMENTS_TITLE}</h1>
           </EuiTitle>
         </EuiPageHeaderSection>
+        {myRole.canManageEngineDocuments && !isMetaEngine && (
+          <EuiPageHeaderSection>
+            <DocumentCreationButton />
+          </EuiPageHeaderSection>
+        )}
       </EuiPageHeader>
-      <EuiPageContent>
-        <EuiPageContentBody>
```

Comment:
Just to confirm, we're purposely getting rid of the `EuiPageContent`/`EuiPageContentBody` wrappers, correct? (no strong feelings either way as I think we don't want the panel appearance, just want to make sure it's intentional)

Reply:
Yes, this was an intentional change to remove the panel appearance I believe. Thank you for checking.